core/src/cpu/event.rs

```diff
@@ -1,7 +1,64 @@
+use std::array;
+
```

Comment:
nit: CpuEvent should be the first thing in this file, not LookupIds given the name of the file is event.rs